policyengine_us/tests/policy/baseline/gov/hhs/medicaid/eligibility/is_medicaid_eligible.yaml

```diff
@@ -71,3 +71,10 @@
     state_code_str: CO
   output:
     is_medicaid_eligible: false
+
+- name: Eligible via California FFYP program
```

Comment:
could you add a couple more tests with all three eligibility criteria to show how they interact?